dvl_gazebo/worlds/uuv_dave_ocean_waves_nogravity.world

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" ?>
```

Comment:
Could you use one of the world files from the uuv_simulator package?  I believe you are using the plugin from that package, so it shouldn't introduce a dependency.

Reply:
Probably.  I used the one from Bruce's demo with a slight modification to move the camera and disable gravity (since the DVL isn't attached to anything, it just freefalls into the mud).  I can probably do the same thing by making it static since I'm manually moving it anyway, but I'll have to check.